### PR TITLE
tweaked form and added link to homepage

### PR DESCRIPTION
**Pull request by malawi-mike**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -149,14 +165,30 @@ class="btn-large waves-effect waves-light lighten-1" style="background-color:rgb
...
-
-		<div class="parallax">
+			
+	</div>
+	
+	<div class="row center-align" style="width:100%;padding:5% 4% 0 0">
+        
+         
+         <a href="add_village.php">
+         <div class="col 12 light-green-text" style="width:100%;">
```
Comment (jdepree) [inline index.php]: I think green is the wrong color choice... maybe semi-transparent white?

**Comment by malawi-mike** [comment]
didn't like the look of this design?

Screen Shot 2019-07-17 at 8 45 14 AM

**Comment by malawi-mike** [comment]
also, now the mobile view is not vertically centered

Screen Shot 2019-07-17 at 8 50 13 AM

**Comment by jdepree** [comment]
Before, the height of that picture was hardcoded for mobile, so whitespace and the title were visible below it.  This doesn't seem desirable.  We could change it back, or find a way to provide dynamic vertical centering (as opposed to hardcoded padding values).

**Comment by jdepree** [comment]
I like the single line design better, but as long as it's not green, I don't care all that much.

**Pull request merged**